Update purpleair.markdown to include instructions to get an API key (#27852)
home-assistant/home-assistant.github.io: source/_integrations/purpleair.markdown

  description: Untrack a sensor.
{% endconfiguration_basic %}

## Creating an API key

> To add this integration you will need a (free) Purple Air API Key.  Detailed instructions are at [https://community.purpleair.com/t/creating-api-keys/3951]([https://community.purpleair.com/t/creating-api-keys/3951]) but in summary you:

Reviewer: It might be worth noting that while a new account comes with 1M free points, after that users will need to buy additional points to continue to use the API. As currently written, the current plugin uses ~30K points/day, so the 1M points will last about a month before the user runs into API errors unless they buy more points. See #93428 for details.

> To add this integration you will need a (free) Purple Air API Key.  Detailed instructions are at [https://community.purpleair.com/t/creating-api-keys/3951]([https://community.purpleair.com/t/creating-api-keys/3951]) but in summary you:

Reviewer 2: ```suggestion
To add this integration, you need a (free) Purple Air API Key. A new account currently comes with 1 million free points. After that, you need to buy additional points to continue to use the API. The current plugin uses ~30K points/day. The 1 million points last about a month. A lack of points will trigger API errors until you buy more points.

Detailed instructions are at [https://community.purpleair.com/t/creating-api-keys/3951]([https://community.purpleair.com/t/creating-api-keys/3951]) but in summary you:
```

> * Create an account at https://develop.purpleair.com/ (which uses Single Sign-On through a Google account)

Reviewer 2: ```suggestion
* Create an account at https://develop.purpleair.com/ (which uses Single Sign-On through a Google account).
```

> * On the ['keys'](https://develop.purpleair.com/keys) page press the "plus" button to create an API key.  Leave the defaults of "Read" and "Enabled".

Reviewer 2: ```suggestion
* On the ['keys'](https://develop.purpleair.com/keys) page press the "plus" button to create an API key.  Leave the defaults of **Read** and **Enabled**.
```
We use bold to markup UI elements.

> * Go to the ['projects'](https://develop.purpleair.com/projects) page and select edit (pencil) button on the listed Project and add points (eg, 1,000,000) then click "Update"

Reviewer 2: ```suggestion
* Go to the ['projects'](https://develop.purpleair.com/projects) page and select the edit (pencil) button on the listed Project.  Add points (for example 1,000,000), then select **Update**.
```

* Go back to the ['keys'](https://develop.purpleair.com/keys) page and copy the API key.  It will be a value like XXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXXX.

> * Paste that API key into the "API Key" field when creating the Integration in Home Assistant

Reviewer 2: ```suggestion
* Paste that API key into the **API Key** field when creating the integration in Home Assistant.
```

> Note that if you are using your own sensor it will need to be set to Public to be visible.

Reviewer 2: ```suggestion
Note that if you are using your own sensor, it will need to be set to **Public** to be visible.
```
